### ISSUE TITLE
re-asses django-braces requirement

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2709]
Django-braces "django-braces 1.13.0 Reusable, generic mixins for Django" may be an inclusion from a prior experiment. This issue is to assess if we actually use this library any longer, and if so for what. The project itself is far from regularly maintained but does seem to be just a set of convinience classes. And as such we likely want to drop this dependency and define just the helper classes that we actually/use need.

https://django-braces.readthedocs.io/en/latest/index.html

https://github.com/brack3t/django-braces/#notes:
django-braces is stable and time-tested. It does not receive a lot of updates and is not in active development.

Also note that the linked changelog does not include PyPi releases:
https://django-braces.readthedocs.io/en/latest/changelog.html

https://pypi.org/project/django-braces/

It is proposed that we drop this dependency as it may well hinder our ability to update freely in the near future.

**Comment by phillxnet** [comment]
```
grep -R "braces"

poetry.lock:name = "django-braces"
poetry.lock:django-braces = [
poetry.lock:    {file = "django-braces-1.13.0.tar.gz", hash = "sha256:ba68e98b817c6f01d71d10849f359979617b3fe4cefb7f289adefddced092ddc"},
poetry.lock:    {file = "django_braces-1.13.0-py2.py3-none-any.whl", hash = "sha256:a457d74ea29478123c0c4652272681b3cea0bf1232187fd9f9b6f1d97d32a890"},
pyproject.toml:django-braces = "==1.13.0"  # look to 1.14.0 (30 Dec 2019) as Django 1.11.0+ now
```
so no code entries: only package related entries.

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2711
